.env.staging.example

```
@@ -119,11 +119,32 @@ VITE_SUPABASE_URL=https://<staging-ref>.supabase.co
+# #VERIFY: confirm CYO_ADVENTURE_OPENROUTER_MODEL below is actually the pinned
+# cheap id and not inherited from the production default before enabling any
+# automated staging generation run.
+#
+# Staging now REQUIRES an OPENROUTER_API_KEY; it did not before. Use a
+# staging-scoped key with its own spend limit, never the production key.
+# -----------------------------------------------------------------------------
+CYO_ADVENTURE_GENERATION_PROVIDER=openrouter
+CYO_ADVENTURE_OPENROUTER_MODEL=deepseek/deepseek-v4-flash
+# The cascade's second leg is never model-overridden, so pin the fallback too or
+# a failover silently lands on the expensive production default.
+CYO_ADVENTURE_OPENROUTER_FALLBACK_MODEL=deepseek/deepseek-v4-flash
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CYO_ADVENTURE_OPENROUTER_FALLBACK_MODEL key should go before the CYO_ADVENTURE_OPENROUTER_MODEL key

(UnorderedKey)